tests/integration_snapshots/test_trace_stats_tracestats.json

```diff
@@ -7,13 +7,418 @@
         "Name": "http.request",
         "Resource": "/users/view",
         "Type": null,
+        "HTTPStatusCode": 0,
         "Synthetics": false,
         "Hits": 5,
         "TopLevelHits": 5,
-        "Duration": 293000,
+        "Duration": 110000,
         "Errors": 1,
-        "OkSummary": 1046,
-        "ErrorSummary": 1046
+        "OkSummary": {
+          "mapping": {
+            "gamma": 1.0156210632401108
+          },
+          "positiveValues": {
+            "contiguousBinCounts": [
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
+              0.0,
```

Comment:
it is worth storing as a summation? or like number of elements + summation?

because it is basically X number of buckets, and then the count of each bucket can vary, but the total value and total number of buckets should be consistent?

Reply:
that's a good idea! i'm actually not sure if they would be consistent 🤔 my guess would be that they would be consistent enough for our testing purposes, but not 100%.

cc @anisaraf

Reply:
For sketches we should compare: 

- accuracy of stats. Total Hits/count and p50/95/99/avg latency values. It probably makes sense to allow comparisons to have an acceptable difference if they aren't exactly stable. 

- total memory consumption of the sketch. If this changes it should be triaged. 

These two should implicitly capture the outcome of various parameters set on the sketch as-well.